### ISSUE TITLE
Reveal.js keyboard navigation is broken

### ISSUE THREAD

**Issue by asajeffrey** [servo/servo#26252]
Point servo at https://revealjs.com (or any other site using reveal) and keyboard navigation doesn't work. The culprit is https://github.com/hakimel/reveal.js/blob/00b0ace3863952ce95df4430d203686fd27bfc20/js/reveal.js#L5233 which checks for editable elements using `document.activeElement.contentEditable !== 'inherit'` rather than `document.activeElement.iscontentEditable`. With that change, reveal works.

Related: https://github.com/servo/servo/issues/12776

**Comment by asajeffrey** [comment]
Fixed upstream by https://github.com/hakimel/reveal.js/issues/2650 and a workaround is #26253.
